### DISCUSSION TITLE
Privacy Settings in the paid version

### DISCUSSION THREAD

**Discussion by Cattes** [community/community#19144] [Copilot Conversations] [answered]
Will we be able to **opt-out of github copilot sending our private, proprietary code to Github / OpenAI**?

This is prevents me from using github copilot. 

I am happy to pay 10$ for it, but I can not use it if it sends out proprietary company data to a third party.

Related Issue: https://github.com/github-community/community/discussions/7163

**Comment by madebyfabian** [comment, marked as answer]
Yes, there is an option to opt out.
<img width="868" alt="Bildschirmfoto 2022-06-22 um 09 56 19" src="https://user-images.githubusercontent.com/32813692/174975471-9171f494-52eb-41e3-84f5-8c15404a24e5.png">

**Reply by youtux** [reply]
how can I access this page again once I submitted my choice?

**Reply by MicroPanda123** [reply]
> 

![obraz](https://user-images.githubusercontent.com/19224202/175089046-60380ff2-97ef-4f54-a517-9ab232f7cfd9.png)
